### PR TITLE
update examples of DOM

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

connect sidebar and update content

### CODE REVIEW

File: files/zh-cn/web/api/document_object_model/examples/index.md [hunk 1]
```diff
@@ -1,378 +1,416 @@
...
 ```
 
-## 示例 6: getComputedStyle
+## 示例 6：getComputedStyle
 
-这个例子演示了如何用{{domxref("window.getComputedStyle")}}方法来获取一个元素的样式而不是使用 style 属性或 JavaScript 的（例如，elt.style.backgroundColor="RGB(173，216，230)"）。列举在 [DOM CSS Properties List](/zh-CN/docs/Web/CSS/Reference) 后面的类型的样式可以用更直接{{domxref("element.style", "elt.style")}} 属性获取。
+这个例子演示了如何使用 {{domxref("window.getComputedStyle")}} 方法来获取一个元素的样式，这些样式不是用 `style` 属性或用 JavaScript 设置的（例如 `elt.style.backgroundColor="rgb(173, 216, 230)"`）。后面这些类型的样式可以用更直接的 {{domxref("HTMLElement.style", "elt.style")}} 属性来检索，其属性在 [DOM CSS 属性列表](/zh-CN/docs/Web/CSS/Reference)中列出。
 
-`getComputedStyle()` 返回了一个 `ComputedCSSStyleDeclaration` 对象，其独立的样式属性可以用该对象的 getPropertyValue() 方法引用，如同下面的例子一样：
+`getComputedStyle()` 返回一个 {{domxref("CSSStyleDeclaration")}} 对象，其各个样式属性可以用这个对象的 {{domxref("CSSStyleDeclaration.getPropertyValue()", "getPropertyValue()"}} 方法进行引用，正如下面的示例文档所示。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document_object_model/examples/index.md:243]:
```suggestion
`getComputedStyle()` 返回一个 {{domxref("CSSStyleDeclaration")}} 对象，其各个样式属性可以用这个对象的 {{domxref("CSSStyleDeclaration.getPropertyValue()", "getPropertyValue()")}} 方法进行引用，正如下面的示例文档所示。
```

File: files/zh-cn/web/api/document_object_model/examples/index.md [hunk 2]
```diff
@@ -1,378 +1,416 @@
...
-本章介绍提供了一些长例子来介绍如何使用 DOM 进行 Web 以及 XML 开发。在可能的情况下，例子只使用普通 API，技巧以及 JavaScript 模式来操作文档对象。
+{{DefaultAPISidebar("DOM")}}
 
-## 示例 1: 高度和宽度
+本章提供了一些介绍如何使用 DOM 进行 Web 以及 XML 开发的较长示例。在可能的情况下，示例只使用 JavaScript 中较常见的 API、技巧以及模式来操作文档对象。
 
-下面的例子展示了在不同尺寸的图片时使用其 height 和 width 属性的情况：
+## 示例 1：高度和宽度
+
+下面的示例展示了在不同尺寸的图片时使用其 `height` 和 `width` 属性的情况：
```
Comment (yin1999) [inline files/zh-cn/web/api/document_object_model/examples/index.md:12]:
```suggestion
下面的示例展示了不同尺寸的图片对 `height` 和 `width` 属性的使用：
```

File: files/zh-cn/web/api/document_object_model/examples/index.md [hunk 3]
```diff
@@ -1,378 +1,416 @@
...
+    </form>
+  </body>
 </html>
 ```
 
-## 示例 3: 操作样式
+## 示例 3：修改样式
 
-在下面这个简单的例子中，使用的 element(DOM 元素) 的 style 对象即对象的 CSS 样式属性来获取一个 HTML 段落元素的一些基本样式属性，DOM 可以检索和设置 CSS 样式。在本例中，你可以直接控制单个样式属性。在下一个的例子里（见例 4），你可以使用 stylesheets 对象 (样式表) 及其 cssRules 对象改变整个文档的样式。
+在这个简单的例子中，HTML 段落元素的一些基本样式属性是使用该元素上的样式对象和该对象的 CSS 样式属性来访问的，这些对象可以从 DOM 中检索和设置。在这个示例中，你是直接操作各个样式的。在下一个例子中（见例 4），你可以使用样式表及其规则来改变整个文档的样式。
```
Comment (yin1999) [inline files/zh-cn/web/api/document_object_model/examples/index.md:121]:
```suggestion
在这个简单的例子中，HTML 段落元素的一些基本样式属性是使用该元素上的样式对象和该对象的 CSS 样式属性来访问的，这些对象可以从 DOM 中检索和设置。在这个示例中，你是直接操作各个样式的。在下一个例子中（见示例 4），你可以使用样式表及其规则来改变整个文档的样式。
```

**Review by yin1999 — APPROVED** [review]
Thanks. Just some translation suggestions

**Pull request merged**